tests/IntegrationTestCase.php

```diff
@@ -2,18 +2,20 @@
 
 namespace DTL\Extension\Fink\Tests;
 
-use PHPUnit\Framework\TestCase;
+use Amp\PHPUnit\AsyncTestCase;
 use Phpactor\TestUtils\Workspace;
 
-class IntegrationTestCase extends TestCase
+class IntegrationTestCase extends AsyncTestCase
```

Comment:
What does `AsyncTestCase` add?

Reply:
It allows yielding promises directly in the test methods, without a custom `Amp\Promise\wait` or `Loop::run()`.